contracts/DepositManager.sol

```diff
@@ -83,9 +95,13 @@ contract DepositCore is SubtreeQueue {
             i >>= 1;
         }
         babyTreesLength = len;
+        // TODO Is there a better way we can mirror this as it is in enqueue function?
+        subtreeID = back + 1;
```

Comment:
Is there a better way we can do this to reduce duplication of next subtreeID being calculated?

Reply:
Yes, let me reply in other comments